test/client/fixtures/sandbox/node/window-test.js

```diff
@@ -16,6 +17,55 @@ test('window.onerror setter/getter', function () {
     strictEqual(getProperty(window, 'onerror'), handler);
 });
 
+if (featureDetection.hasUnhandledRejectionEvent) {
+    module('window.onunhandledrejection', function () {
+        asyncTest('must be overriden (GH-1247)', function () {
```

Comment:
Add test for case then page has handler for `window.onunhandledrejection`. In this case we should not raise error.